### ISSUE TITLE
Create sequence before table

### ISSUE THREAD

**Issue by make** [kaleidos/grails-postgresql-extensions#100]
I am using following domain class mapping to enable direct SQL inserts.

```groovy
class Foo {
    static mapping = {
        id defaultValue:"nextval('seq_foo')"
    }
}
```

When using dbCreate = 'update' in datasource table and sequence are automatically created when running 'grails run-app'.
The problem is that table is created first and causes error:
```
[main] ERROR o.h.tool.hbm2ddl.SchemaUpdate - ERROR: relation "seq_foo" does not exist
```

The workaround is to rerun "grails run-app".

Sequence should be created before table to fix that issue.

**Comment by ilopmar** [comment]
Maybe I'm missing something but why are you doing this? When you say _to enable direct SQL inserts_ you mean to be able to do an insert *without* setting a value for the `id` column?

**Comment by make** [comment]
Yes. I want to enable both manual SQL inserts and GORM inserts using the same auto-incrementing id sequence.

So mapping ```id defaultValue:"nextval('seq_foo')"``` should be declared to enable following to work
```groovy
new Sql(dataSource).executeInsert('INSERT INTO foo DEFAULT VALUES')
```

Equal GORM insert is
```groovy
new Foo().save(flush:true)
```
